pkg/driver/driver_gcp.go

```diff
@@ -329,3 +329,17 @@ func extractProject(serviceaccount []byte) (string, error) {
 	}
 	return j.Project, nil
 }
+
+// GetVolNames parses volume names from pv specs
+func (d *GCPDriver) GetVolNames(specs []corev1.PersistentVolumeSpec) ([]string, error) {
+	names := []string{}
+	for _, spec := range specs {
```

Comment:
Same shallow copy issue as [above](https://github.com/gardener/machine-controller-manager/pull/275/files#diff-face99aebd3cd79577003f7df747c863R335) but with less impact because the number of volumes per pod will be less.